### ISSUE TITLE
Meet a problem when updating VS2017 to 15.9.3

### ISSUE THREAD

**Issue by lgjie** [MicrosoftDocs/visualstudio-docs#2021]
Error Information
AppInsightsEvent Name = vs/setup/bootstrapper/error
          Description = Failure to download
     TelemetrySession = '501af9d3-297a-401b-bddf-203e88bd0775' Started = True OptIn=True IsInitialized = True Cloned = False
      WatsonEventType = VisualStudioNonFatalErrors2
             UTC time = 2018-12-07T06:18:30

 Exception:
 Microsoft.VisualStudio.Setup.Security.InvalidSignatureException
 Package signature verification threw exception: 终结点映射器中没有更多的终结点可用。
 , for path: vs_installer.opc"
 Microsoft.VisualStudio.Setup.Download.DownloadManager.DownloadWithRetry(IDownloadEngine[] engines, Uri uri, Stream outputStream, ProgressUpdateCallback progress, CancellationToken cancellationToken, DownloadContext downloadContext, Boolean verifySignature)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: deb3386e-6b18-1cbd-9f1b-523b27b6521b
* Version Independent ID: 5ddf5e28-86d1-28b6-80ef-332bf93e17ac
* Content: [Update Visual Studio 2017 - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/install/update-visual-studio?view=vs-2017)
* Content Source: [docs/install/update-visual-studio.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/install/update-visual-studio.md)
* Product: **visual-studio-dev15**
* GitHub Login: @TerryGLee
* Microsoft Alias: **tglee**

**Comment by gewarren** [comment]
Thanks! Closing this issue.

**Comment by lgjie** [comment]
@gewarren, I've resolved this issue, the solution is update windows OS to the latest version and then go to update VS2017, everything is ok now. BTW, if the OS self update function doesn't work, we can go to https://www.microsoft.com/zh-cn/software-download/windows10 for download the latest version of update package and install it. I tried the latter one, it works and every weird problem met before has gone.
